platforms/documentation/docs/src/docs/userguide/best-practices/bp_dependencies.adoc

```diff
@@ -15,4 +15,41 @@
 [[best_practices_for_dependencies]]
 = Best Practices for Dependencies
 
-Add content here.
+[[dont_depend_on_kotlin_stdlib]]
+== Don't Explicitly Depend on the Kotlin Standard Library
+:keywords: dependencies
+
+Kotlin Gradle Plugin automatically adds a dependency on the Kotlin standard library (`stdlib`) to each source set, so there is no need to add it explicitly.
+
+=== Explanation
+
+The version of the standard library added is the same as the version of the Kotlin Gradle Plugin.
+If your build does not depend on a specific or different version of the standard library then an explicit dependency is not needed and shouldn't be added.
+
+Note that setting the `kotlin.stdlib.default.dependency` property to `false` prevents the Kotlin plugin from automatically adding the Kotlin standard library dependency to your project which can be useful in specific scenarios, such as when you want to manage the Kotlin standard library dependency version manually.
```

Comment:
I would consider even removing the second sentence here because it's obvious/redundant:

```suggestion
NOTE: Setting the `kotlin.stdlib.default.dependency` property to `false` prevents the Kotlin plugin from automatically adding the Kotlin standard library dependency to your project. This can be useful in specific scenarios, such as when you want to manage the Kotlin standard library dependency version manually.
```